processor/src/handler.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 Bitwise IO
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use sawtooth_sdk::messages::processor::TpProcessRequest;
+use sawtooth_sdk::processor::handler::ApplyError;
+use sawtooth_sdk::processor::handler::TransactionContext;
+use sawtooth_sdk::processor::handler::TransactionHandler;
+
+use bond_common::addressing::*;
+
+const BOND_NAMESPACE: &str = "bond namespace";
```

Comment:
As a suggestion, because you are importing everything from `bond_common::addressing`, you can just get the bond namespace that is defined there. So you can use `FAMILY_NAMESPACE` to get the bond namespace and don't have to redefine it here. You might need to make FAMILY_NAMESPACE in `addressing` public.